redaxo/src/core/lib/packages/manager.php

```diff
@@ -478,9 +478,26 @@ public function checkPackageRequirement($packageId)
             if ('' != $requirements['packages'][$packageId]) {
                 $required_version = ' '.$requirements['packages'][$packageId];
             }
-            $this->message = $this->i18n('requirement_error_' . $package->getType(), $packageId.$required_version);
+
+            $installer = rex_addon::get('install');
+
+            if (!rex_package::exists($packageId)) {
+                if ($installer->isAvailable()) {
+                    // package need to be installed
+                    $installUrl = rex_url::backendPage('install/packages/add', ['addonkey' => $packageId]);
+
+                    $this->message = $this->i18n('requirement_error_' . $package->getType(), $packageId.$required_version) . ' <a href="'. $installUrl .'">'. $this->i18n('install_via_installer', $packageId) .'</a>';
+                    return false;
+                }
+
+                $this->message = $this->i18n('requirement_error_' . $package->getType(), $packageId.$required_version);
+                return false;
+            }
+
+            $this->message = $this->i18n('requirement_error_' . $package->getType(), $packageId.$required_version) . ' <a href="#package-'. $packageId .'">'. $this->i18n('install_jump_to_package', $packageId) .'</a>';
```

Comment:
Ich glaube der Part hier greift auch während des Updates im Installer, und der Link passt dann nicht.

Reply:
wie kann ich das testen?

Reply:
sollte man ggf. einfach die url mit aufnehmen, sodass es von jeder Seite heraus funktionieren kann?

Reply:
> wie kann ich das testen?

Gar nicht so leicht. Die Konstellation müsste sein:
Ein Addon updaten, welches bereits installiert ist. Und das Update benötigt neu ein weiteres Addon, was die vorherige Version noch nicht benötigte.

Ggf. kann man es irgendwie simulieren, zum beispiel hier:
https://github.com/redaxo/redaxo/blob/0c6a9302b536400d2ec416a14b2b46b42db5a11e/redaxo/src/addons/install/lib/package/package_update.php#L52
da in `$config['requires']['packages']` noch ein addon einschleusen. und dann ein installiertes addon versuch zu updaten.

Reply:
ich denke ich hab es jetzt "korrekt" mit absoluter url.. habs aber nicht getestet